services/frontend/src/scenes/Profile/Profile.js

```diff
@@ -37,10 +38,22 @@ const dummyData = [
     }
 ];
 
-const Profile = ({projects = dummyData}) => (
-    <ProfileLayout
-        projects={projects}
-    />
+//Split at each word, take the first letter and then convert them to uppercase.
+const addAvatarAcronym = (name) => (
```

Comment:
Yeah, that was my intention initially. Probably do some editing to make it so names like WWW would still work.

That said, I swapped it to first and last since fixing CSS text resizing is a feature that would take a lot longer.